packages/rum-core/src/domain/contexts/displayContext.ts

```diff
@@ -0,0 +1,12 @@
+import { isExperimentalFeatureEnabled } from '@datadog/browser-core'
+
+export function getDisplayContext() {
+  if (!isExperimentalFeatureEnabled('clickmap')) return
+
+  return {
+    viewport: {
+      width: window.innerWidth,
+      height: window.innerHeight,
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Apparently getting the window dimension might force the browser to recompute the layout (cf list [here](https://gist.github.com/paulirish/5d52fb081b3570c81e3a)). Maybe listen for `resize` event and cache its value?

Reply:
Indeed, good.  Do you think we should do the same [here](https://github.com/DataDog/browser-sdk/blob/336ebe7c6cd3dc9c87499cbf9cac6bb5e8fe6da8/packages/rum/src/domain/record/observer.ts#L223-L234)

Reply:
that's already what we do :) (but also we throttle the listener)

Reply:
Nice!